django/db/models/query.py

```diff
@@ -563,9 +563,12 @@ def bulk_update(self, objs, fields, batch_size=None):
                     case_statement = Cast(case_statement, output_field=field)
                 update_kwargs[field.attname] = case_statement
             updates.append(([obj.pk for obj in batch_objs], update_kwargs))
+        rows_updated = 0
```

Comment:
(I would have thought `updated` was enough here, but I guess with another variable called `updates` above it's probably worth leaving this as you have it to avoid any potential confusion.)